application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -92,12 +96,20 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalChannelId = event.getTarget().getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
         String mostCommonTag = tags.get(0);
+        String chatGptPrompt = String.format(
+                "Generate topic for question: %s. Minimum %s and maximum %s characters.",
+                originalMessage, TITLE_MIN_LENGTH, TITLE_MAX_LENGTH);
+        Optional<String> chatGptResponse = chatGptService.ask(chatGptPrompt, mostCommonTag);
```

Comment:
your earlier approach of not passing any tags was right since i just realised it's directly transfering a text so it's impossible to know what tag would be here. My bad for suggesting you otherwise, instead of mostCommonTag you can pass empty string 👍

Reply:
I had thought the same thing but I thought that you guys must rely on the config tags to get responses related to the topic. I should have clarified it with you.